homeassistant/components/pi4ioe5v9xxxx/binary_sensor.py

```diff
@@ -0,0 +1,115 @@
+"""Support for binary sensor using RPi GPIO."""
+import logging
+
+import voluptuous as vol
+
+from smbus2 import SMBus  # pylint: disable=import-error
+from smbus2 import i2c_msg  # pylint: disable=import-error
+
+from homeassistant.components.binary_sensor import PLATFORM_SCHEMA, BinarySensorDevice
+from homeassistant.const import DEVICE_DEFAULT_NAME
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_INVERT_LOGIC = "invert_logic"
+CONF_PINS = "pins"
+CONF_I2CBUS = "i2c_bus"
+CONF_I2CADDR = "i2c_address"
+CONF_BITS = "bits"
+
+DEFAULT_INVERT_LOGIC = False
+DEFAULT_BITS = 24
+_I2C_ADDR = 0x20
+
+
+_SENSORS_SCHEMA = vol.Schema({cv.positive_int: cv.string})
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_PINS): _SENSORS_SCHEMA,
+        vol.Required(CONF_I2CBUS): cv.positive_int,
+        vol.Optional(CONF_I2CADDR, default=_I2C_ADDR): cv.positive_int,
+        vol.Optional(CONF_BITS, default=DEFAULT_BITS): cv.positive_int,
+        vol.Optional(CONF_INVERT_LOGIC, default=DEFAULT_INVERT_LOGIC): cv.boolean,
+    }
+)
+
+_PORT_VALUE = [0xFF]
+_BUS = None
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the IO expander devices."""
+    global _PORT_VALUE
+    global _I2C_ADDR
+    global _BUS
+    invert_logic = config.get(CONF_INVERT_LOGIC)
```

Comment:
```suggestion
    invert_logic = config[CONF_INVERT_LOGIC]
```

Don't use get, since it has a value as listed in the schema.

This needs to be applied to all other places as well.
Only use the `get()` method when the configuration parameter is optional and doesn't have a default value set in the schema.

Reply:
Done.